### DISCUSSION TITLE
Do HYBRID type servers support SPIGET_RESOURCES environment?

### DISCUSSION THREAD

**Discussion by aruneko** [itzg/docker-minecraft-server#2281] [Q&A] [answered]
I have noticed that the hybrid type servers are ignoring SPIGET_RESOURCES environment variable when I deployed a mohist server.

Hybrid servers call `start-setupWorld` script [here](https://github.com/itzg/docker-minecraft-server/blob/7f7636e85e3a56df4ac1fd48f40beb2f6b96246f/scripts/start-deployMohist#L43) instead of calling `start-spiget` script. In my opinion, we can call `start-spiget` script here in hybrid type servers to download not only mods and also plugins.

For automation, it is convenience that supporting SPIGET_RESOURCES in hybrid type servers. How do you think about it?

**Comment by itzg** [comment]
I'll convert this to an issue. I didn't intend for hybrid types to skip spiget support.

**Comment by itzg** [comment, marked as answer]
Latest image now supports SPIGET_RESOURCES for hybrid server types.

**Reply by aruneko** [reply]
Thank you, it works well!

**Comment by aruneko** [comment]
resolved by #2282
